src/transformers/models/bert/tokenization_bert_tf.py

```diff
@@ -0,0 +1,220 @@
+import os
+from typing import List, Union
+
+import tensorflow as tf
+
+from tensorflow_text import FastBertTokenizer, ShrinkLongestTrimmer, case_fold_utf8, combine_segments, pad_model_inputs
+
+from ...utils import requires_backends
```

Comment:
You don't need this anymore since you went the dummy route. The dummy objects will be imported if tf text is not installed and raise the error for you.

Reply:
I don't think I made the dummy objects in the end! I'll leave it in for now, but might switch to dummy objects when we add a few more tokenizers

Reply:
This is not resolved yet, please do address :-)

Reply:
Ah, sorry!

Reply:
@sgugger I dug around in this one for a little bit - I figured out the reason I had trouble with the original approach with dummies is a missing `_` in one of the regexes in `check_dummies.py`, which caused it to miss `is_tensorflow_text_available`. Once that was fixed I was able to create `dummy_tensorflow_text_objects.py` without problems and remove `requires_backends` in the class `__init__()`, so hopefully everything is okay now! If you could double-check it to make sure I didn't miss anything, because I'm still a little uncertain about adding a new set of dummy objects like this, that'd be great!

Reply:
That works for me!